PHP/PHP Source.sublime-syntax

```diff
@@ -208,65 +233,102 @@ contexts:
           scope: punctuation.section.array.end.php
           pop: true
         - include: expressions
-    - match: (?i)\s*\(\s*(array|real|double|float|int(eger)?|bool(ean)?|string|object|binary|unset)\s*\)
-      captures:
-        1: storage.type.php
-    - match: (?i)\b(array|real|double|float|int(eger)?|bool(ean)?|string|class|clone|function|interface|object)\b
-      scope: storage.type.php
-    - match: (?i)\b(parent|self)\b
-      scope: variable.language.php
-    - match: (?i)\b(global|abstract|const|extends|implements|final|p(r(ivate|otected)|ublic)|static)\b
-      scope: storage.modifier.php
-    - include: object
-    - match: ;
-      scope: punctuation.terminator.expression.php
-    - include: heredoc
-    - include: numbers
-    - match: (\+|-|\*|/|%|&|\||\^|>>|<<|\.|\?\?)=
-      scope: keyword.operator.assignment.augmented.php
-    - match: "=>"
-      scope: keyword.operator.key.php
-    - match: '(?:(\=)(&))|(&(?=[[:alpha:]$_]))'
-      captures:
-        1: keyword.operator.assignment.php
-        2: storage.modifier.reference.php
-        3: storage.modifier.reference.php
-    - match: (@)
-      scope: keyword.operator.error-control.php
-    - match: (\-\-|\+\+)
-      scope: keyword.operator.increment-decrement.php
-    - match: (\-|\+|\*|/|%)
-      scope: keyword.operator.arithmetic.php
-    - match: '&&|\|\|'
-      scope: keyword.operator.logical.php
-    - match: '<<|>>|~|\^|&|\|'
-      scope: keyword.operator.bitwise.php
-    - match: (===|!==|<=>|==|!=|<=|>=|<>|<|>)
-      scope: keyword.operator.comparison.php
-    - match: (?i)(!)|\b(and|or|xor|as)\b
-      scope: keyword.operator.logical.php
-    - include: function-call
-    - match: \.\.\.
-      scope: keyword.operator.spread.php
-    - match: \.
-      scope: keyword.operator.string.php
-    - match: "="
-      scope: keyword.operator.assignment.php
-    - match: '(?i)\b(instanceof)\s+(?=[[:alpha:]\\$_])'
-      captures:
-        1: keyword.operator.type.php
-      push:
-        - match: '(?=[^[:alnum:]_\\$])'
-          pop: true
-        - include: class-name
-        - include: variables
-    - include: instantiation
-    - match: '(?i)(goto)\s+({{identifier}})'
-      captures:
-        1: keyword.control.goto.php
-        2: support.other.php
-    - include: string-backtick
-    - match: '(\[)\s*(\])'
+        - match: (?i)\s*\(\s*(array|real|double|float|int(eger)?|bool(ean)?|string|object|binary|unset)\s*\)
```

Comment:
the indentation is wrong here.

Reply:
Yeah, that's the problem I get when I copy paste the expanded regex from what is generated by the script 😞 I'll sort it out when I'm at my desktop, it's a little difficult to identify the issues on such a small screen.